tools/rsdl/dotnet/README.md

```diff
@@ -9,10 +9,10 @@ The rapid cli program can be run from this folder with
 
 ```sh
 cd rapid
-dotnet run -- sample.rsdl --format JSON
-dotnet run -- sample.rsdl --format XML
-dotnet run -- sample.rsdl --format XML.JSON
-dotnet run -- sample.rsdl
+dotnet run -- samples/sample.rsdl --format JSON
+dotnet run -- samples/sample.rsdl --format XML
+dotnet run -- samples/sample.rsdl --format XML.JSON
+dotnet run -- samples/sample.rsdl
```

Comment:
@chrisspre These commands currently fail with

```
Unhandled exception. System.InvalidCastException: Unable to cast object of type 'Microsoft.OData.Edm.EdmNavigationProperty' to type 'Microsoft.OData.Edm.IEdmStructuralProperty'.
   at rapid.rsdl.ModelBuilder.<>c__DisplayClass8_0.<AddStructuredType>b__0(RdmProperty key) in C:\SAPDevelop\odata-rapid\tools\rsdl\dotnet\rapid.rdm.transformation\ModelBuilder.cs:line 117
```

Reply:
This turns out to be a fundamental bug in the way the transformation resolves type references.
Started to investigate this in PR #166 
Is not related to this PR and I would prefer to handle it separately.